src/update.js

```diff
@@ -0,0 +1,3 @@
+export default function setAsCompleted(todo) {
+  todo.completed = !todo.completed;
+}
```

Comment:
There is no need to create a hole module to add this little function.
Instead, you can create a module called utils.js and add all functions related to the new todo task like add, Remove, Edit, and delete completed and import this module to index.js